crates/proof-of-sql-parser/src/intermediate_ast_tests.rs

```diff
@@ -349,7 +349,7 @@ fn we_can_parse_a_query_with_one_equals_filter_having_a_negative_literal() {
 }
 
 #[test]
-fn we_can_parse_a_query_with_one_not_equals_filter_expression() {
+fn we_can_parse_a_query_with_one_not_equals_filter_execession() {
```

Comment:
This is renamed wrong. Double check everything.

Reply:
@JayWhite2357 Sorry..fixed.